ci/run_cudf_ctests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+if [ -d "${INSTALL_PREFIX:-${CONDA_PREFIX:-/usr}}/bin/gtests/libcudf/" ]; then
```

Comment:
None of these tests should ever be skipped. If this directory doesn't exist, we should fail (and not skip this block). I think I would just remove the conditionals everywhere.

Reply:
Sure, I can remove these checks.